src/styles/blocks/_watch.scss

```
@@ -0,0 +1,155 @@
+.watch {
+  @extend %flex-center;
+  align-items: center;
+  height: 100vh;
+  background-color: $bgr-color;
+
+  &__body {
+    @extend %flex-center;
+    position: relative;
+    background-color: $watch-bgr-color;
+    height: $watch-height;
+    width: $watch-width;
+    border-radius: 50%;
+    border: 5px solid rgb(255, 255, 255);
+    text-shadow: $ts-watch-body;
+    box-shadow: $bs-watch-body;
+
+    @media (max-width: 660px) {
+      height: $watch-height/1.6;
+      width: $watch-width/1.6;
+    }
+  }
+
+  &__number {
+    display: flex;
+    position: absolute;
+    color: #fff;
+    left: 45%;
+    top: -14%;
+    font-size: $fz-number;
+    text-shadow: $ts-number;
+    transition: 1s;
+    animation-name: test;
+    animation-duration: 5000ms;
+    animation-iteration-count: infinite;
+    animation-timing-function: linear;
```

```suggestion
    animation: test, 5s, infinite, linear;
```
Это можно в одну строку записать

---

Це зайві стилі - видалив